remix-analyzer/src/solidity-analyzer/modules/deleteDynamicArrays.ts

```diff
@@ -5,8 +5,8 @@ import { AnalyzerModule, ModuleAlgorithm, ModuleCategory, ReportObj, Compilation
 
 export default class deleteDynamicArrays implements AnalyzerModule {
   rel: UnaryOperationAstNode[] = []
-  name: string = `Delete on dynamic array: `
-  description: string = `Use require/assert appropriately`
+  name: string = `Delete dynamic array: `
+  description: string = `Use require/assert to ensure`
```

Comment:
to ensure what ?

Reply:
to ensure the deletion of array

Reply:
actually I think there is a mistake with the description, even for the past commits `use require/assert appropriately` looks like a wrong description.
That one is more: 'using `delete` on a dynamic array might lead to OOG error' (or something equivalent)

Reply:
here the original https://github.com/ethereum/remix/blob/master/remix-analyzer/src/solidity-analyzer/modules/guardConditions.ts#L9

Reply:
Is the issue here ever require/assert?  We had spoke about this - I believe.

Reply:
on of the issue here  is that the desc `Use require/assert to ensure` doesn't describe what the module actually do.